### PR TITLE
add: UI Template choices

### PR DESCRIPTION
**Pull request by rajarju**
Updated docs to include
- New vanilla html/js template choice

### CODE REVIEW

File: getting_started/first_app.md [hunk 1]
```diff
@@ -142,6 +142,13 @@ Note that you may not see all the options listed below on your command line, bec
     - **Adobe Campaign Standard**: including dependencies and examples of accessing the [Adobe Campaign Standard (ACS) API](https://docs.adobe.com/content/help/en/campaign-standard/using/working-with-apis/get-started-apis.html)
     - **Adobe Experience Platform: Realtime Customer Profile**: including dependencies and examples of accessing the [Realtime Customer Profile API of Adobe Experience Platform](https://www.adobe.io/apis/experienceplatform/home/api-reference.html#!acpdr/swagger-specs/real-time-customer-profile.yaml)
     - **Generic**: a generic back-end action with hello world flow that can be reused and modified e.g. for simple serverless computing or 3rd party API integration
+    
+1. If you included `Web Assets` under Adobe I/O App features you will be given two choices. One to include React Spectrum based UI teamplate or a Vanilla HTML/JS one
```
Comment (meryllblanchet) [inline getting_started/first_app.md:146]: teamplate -> template

Could we link `[React Spectrum](https://react-spectrum.adobe.com/)`?

Could we mention that the Vanilla HTML/JS template is also based on `[Spectrum CSS](https://opensource.adobe.com/spectrum-css/)`?

Could we mention that both templates come with the boilerplate code needed to integrate the Project Firefly application within the `[Adobe Experience Cloud](https://github.com/AdobeDocs/project-firefly/blob/master/guides/exc_app/overview.md)`?

**Review by meryllblanchet — CHANGES_REQUESTED** [review]
Thanks for the contribution @rajarju !
I added some comments to update the PR.

**Comment by sarahxxu** [comment]
cc @rajarju do you want to update the pr?

**Comment by rajarju** [comment]
@sarahxxu Thankd for the followup 
Updated the PR

**Pull request merged**